src/main/java/mate/academy/hibernate/relations/exeptions/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.hibernate.relations.exeptions;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message, Throwable cause) {
+        super(message);
+    }
```

Comment:
In your `DataProcessingException` constructor with two parameters, you are not passing the `cause` to the `super` constructor. You should pass both `message` and `cause` to properly chain the exception. The correct implementation should look like this: `super(message, cause);`.